portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -33,15 +36,14 @@
 public class LoginStatusServlet extends HttpServlet {
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      response.setContentType("text/html;");
+      response.setContentType("application/json;");
       PrintWriter out = response.getWriter();
 
       UserService userService = UserServiceFactory.getUserService();
-      System.out.println("Receive request: " + userService.isUserLoggedIn());
-      if (userService.isUserLoggedIn()) {
-          out.println("true");
-      } else {
-          out.println("false");
-      }
+      List<Boolean> result = new ArrayList<>();
```

Comment:
This is fine for now, but I think you'll usually want a more structured response so that you can understand what's happening on the other end of the response. For example, if this response was a map like `{loggedIn: true}`, then the JavaScript on the other end would be able to get the data it cares about from `response['loggedIn']` instead of `response[0]`. That helps readers of your front-end code understand what's going on.